test/extended/etcd/helpers/helpers.go

```diff
@@ -365,6 +398,8 @@ func SkipIfUnsupportedPlatform(ctx context.Context, oc *exutil.CLI) {
 	skipIfSingleNode(oc)
 	skipIfBareMetal(oc)
 	skipIfVsphere(oc)
+	skipIfGCP(oc)
+	skipIfOpenStack(oc)
```

Comment:
I am skipping these platforms, still the ci jobs are running and failing :/

Reply:
can you create your "own" `SkipIfUnsupportedPlatform`? We want to run GCP and OpenStack for the other vertical scaling tests...

Reply:
Yeah don't change this. We can't blindly skip on these platforms as it's affecting the other test.
I'm not sure of a better way to do this but in my PR I'm explicitly checking if the cluster CPMS is present or not and then skipping accordingly rather than hard coding platforms.
https://github.com/openshift/origin/pull/27497/files#diff-03b29bf92b33772cc9ec978681227c6c4fdfdca04d2279fac287a5ece33f7a4fR212-R223

So you could also try that.

Reply:
removed 👍🏽

Reply:
waiting for #27497  to be merged, need to reuse https://github.com/openshift/origin/pull/27497/files#diff-03b29bf92b33772cc9ec978681227c6c4fdfdca04d2279fac287a5ece33f7a4fR212-R223